projects/storefrontlib/src/recipes/b2c-storefront.module.ts

```diff
@@ -4,8 +4,7 @@ import {
   provideDefaultConfig,
   provideDefaultConfigFactory,
 } from '@spartacus/core';
-import { CmsLibModule } from '../cms-components/cms-lib.module';
-import { StorefrontConfig } from '../storefront-config';
+import { CmsLibModule, StorefrontConfig } from '@spartacus/storefront';
```

Comment:
Doesn't it cause circullar dependency issue? We should reference filces from the scope of the same library using relative direct paths to files.